ETC_en.tsv

```diff
@@ -460,7 +460,7 @@ ETC_20150317_000459	Strictly speaking, it is a creature of magic whose name wish
 ETC_20150317_000460	Echad
 ETC_20150317_000461	Is a creature of magic that was created to protect the king's tomb. However, it is the stronger the Asmodian also been rice goddess of power.
 ETC_20150317_000462	Shtayim
-ETC_20150317_000463	Despite it's appearance, Creatures of this magic has recieved the blessing of the goddess. Can be fatal to Asmodian.
+ETC_20150317_000463	Despite it's appearance, Creatures of this magic has received the blessing of the goddess. Can be fatal to Asmodian.
```

Comment:
I just fixed the typo.
Will need to check consistency in all files in a future pull about other stuff :-)